src/main/java/org/primefaces/component/dataview/DataView.java

```diff
@@ -182,7 +183,7 @@ public void resetMultiViewState() {
     public void restoreMultiViewState() {
         DataViewState viewState = getMultiViewState(false);
         if (viewState != null) {
-            if (viewState.getLayout() != null && viewState.getLayout().length() > 0) {
+            if (viewState.getLayout() != null && !LangUtils.isValueBlank(viewState.getLayout())) {
```

Comment:
remove the != null, isValueBlank covers that

Reply:
done

Reply:
Check again :wink: